cloud-foundry/nginx/conf/includes/redirect.conf

```diff
@@ -1,3 +1,48 @@
+rewrite ^/kafka-docker-latest$ https://repo.spring.io/libs-release/org/springframework/cloud/stream/app/spring-cloud-stream-app-descriptor/Einstein.SR2/spring-cloud-stream-app-descriptor-Einstein.SR2.stream-apps-kafka-docker redirect;
+rewrite ^/task-maven-latest$ https://repo.spring.io/libs-release/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.RELEASE/spring-cloud-task-app-descriptor-Elston.RELEASE.task-apps-maven redirect;
+rewrite ^/task-docker-latest$ https://repo.spring.io/libs-release/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.RELEASE/spring-cloud-task-app-descriptor-Elston.RELEASE.task-apps-docker redirect;
+
+rewrite ^/Elston-GA-task-applications-maven$ https://repo.spring.io/libs-release/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.RELEASE/spring-cloud-task-app-descriptor-Elston.RELEASE.task-apps-maven redirect;
+rewrite ^/Elston-BUILD-SNAPSHOT-task-applications-maven$ https://repo.spring.io/libs-snapshot-local/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.BUILD-SNAPSHOT/spring-cloud-task-app-descriptor-Elston.BUILD-SNAPSHOT.task-apps-maven redirect;
+rewrite ^/Elston-GA-task-applications-docker$ https://repo.spring.io/libs-release/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.RELEASE/spring-cloud-task-app-descriptor-Elston.RELEASE.task-apps-docker redirect;
+rewrite ^/Elston-BUILD-SNAPSHOT-task-applications-docker$ https://repo.spring.io/libs-snapshot-local/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.BUILD-SNAPSHOT/spring-cloud-task-app-descriptor-Elston.BUILD-SNAPSHOT.task-apps-docker redirect;
+
+rewrite ^/Dearborn-SR1-task-applications-maven$ https://repo.spring.io/libs-release-local/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Dearborn.SR1/spring-cloud-task-app-descriptor-Dearborn.SR1.task-apps-maven redirect;
+rewrite ^/Dearborn-BUILD-SNAPSHOT-task-applications-maven$ https://repo.spring.io/libs-snapshot-local/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Dearborn.BUILD-SNAPSHOT/spring-cloud-task-app-descriptor-Dearborn.BUILD-SNAPSHOT.task-apps-maven redirect;
+rewrite ^/Dearborn-SR1-task-applications-docker$ https://repo.spring.io/libs-release-local/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Dearborn.SR1/spring-cloud-task-app-descriptor-Dearborn.SR1.task-apps-docker redirect;
+rewrite ^/Dearborn-BUILD-SNAPSHOT-task-applications-docker$ https://repo.spring.io/libs-snapshot-local/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Dearborn.BUILD-SNAPSHOT/spring-cloud-task-app-descriptor-Dearborn.BUILD-SNAPSHOT.task-apps-docker redirect;
+
+rewrite ^/Clark-SR1-task-applications-maven$ https://repo.spring.io/libs-snapshot/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Clark.SR1/spring-cloud-task-app-descriptor-Clark.SR1.task-apps-maven redirect;
```

Comment:
Clark can be removed. It will go EOL when Boot 1.x goes EOL in August. It is better not to promote it.